lib/commanded/aggregates/aggregate.ex

```diff
@@ -396,6 +399,24 @@ defmodule Commanded.Aggregates.Aggregate do
     end
   end
 
+  @doc false
+  @impl GenServer
+  def handle_info({:take_snapshot, lifespan_timeout}, %Aggregate{} = state) do
+    state = %Aggregate{state | lifespan_timeout: lifespan_timeout}
+
+    Logger.debug(describe(state) <> " recording snapshot")
```

Comment:
I think it makes sense to leave the logger statement in `do_take_snapshot/1`

Reply:
Agreed, changed in cfef270fb330ab3ba452d6a4d710aa88f4a46af7!